run_dir/design/base.html

```diff
@@ -64,6 +64,9 @@
                       <li><a href="/reads_total/">Read Count Totals</a></li>
                       <li><a href="/proj_meta">Compare Project Meta</a></li>
                       <li><a href="/instrument_logs">View Instrument Logs</a></li>
+                      {% if user.role == 'admin'%}
+                      <li><a href="/assign_roles">Assign roles</a></li>
```

Comment:
I just realised "User Roles" is a better header to use in the menu. Could you change that?

Reply:
Its updated now!